test/id.js

```diff
@@ -44,6 +44,14 @@ api.metatests.case('Common / id', {
     [123456789,        'cd15/075b'],
     [123456789123,     '1a83/be99/001c'],
     [9007199254740991, 'ffff/ffff/ffff/001f']
+  ],
+  'common.pathToId': [
+    ['0000/0000',           0],
+    ['0001/0000',           1],
```

Comment:
Add a few corner cases that make sure that id will be different for '0001/0000' and '0000/0001' and such.